mlflow/types/schema.py

```diff
@@ -160,34 +393,47 @@ def name(self) -> Optional[str]:
 
     @experimental
     @property
-    def optional(self) -> bool:
-        """Whether this column is optional."""
-        return self._optional
+    def required(self) -> bool:
+        """Whether this column is required."""
+        return self._required
```

Comment:
Is this breaking change okay? We marked as @experimental here. Or we raise a deprecation warning first?